### DISCUSSION TITLE
nselect_laqa

### DISCUSSION THREAD

**Discussion by ypanhtu** [Tomoki-YAMASHITA/CrySPY#71] [Q&A]
Dear Dr. Yamashita,

nselect_laqa is the number of candidates one selects at one time.  How to set this parameter?  It it obviously related to the computational efficiency and accuracy？

Thanks a lot.

Roc

HTU

**Comment by Tomoki-YAMASHITA** [comment]
If you have the computing resources to submit multiple jobs at once, it is more efficient to set nselect_laqa to that number.
It depends on your machine.
